takwimu/takwimu_ui/src/Theme.js

```diff
@@ -6,6 +6,8 @@ const FONT_FAMILY_TEXT = '"Muli", sans-serif';
 const Theme = createMuiTheme({
   palette: {
     primary: { main: '#29a87c' },
+    secondary: { main: '#ffffff' },
+    text: { primary: '#fff', secondary: '#000' },
```

Comment:
This messes up other components... primary should be black.
![Screenshot from 2019-03-26 10-38-47](https://user-images.githubusercontent.com/1779590/54979194-9770c080-4fb3-11e9-9182-55770cfd5f3c.png)